### DISCUSSION TITLE
Error while importing foundry devops

### DISCUSSION THREAD

**Discussion by Abhishek-2416** [Cyfrin/foundry-full-course-cu#1099] [Q&A]
When i write forge coverage 
It throws an error Error: 
Failed to resolve file: "/home/abhishekalimchandani/Desktop/Projects/Patrick Collins/Foundy NFTs/../lib/src/DevOpsTools.sol": No such file or directory (os error 2).
 Check configured remappings..
    --> "/home/abhishekalimchandani/Desktop/Projects/Patrick Collins/Foundy NFTs/script/Interaction.s.sol"
        "foundry-devops/src/DevOpsTools.sol"

It is in the correct location still it shows this what do i do
Here is my code https://github.com/Abhishek-2416/Foundry-NFTs

**Comment by 0xusmanf** [comment]
You are not remapping `foundry-devops` correctly, your remapping in `foundry.toml` should look like below.
```toml
remappings = ["@openzeppelin/contracts=lib/openzeppelin-contracts/contracts","foundry-devops=lib/foundry-devops"]
```

Correct this and your code will work fine.

**Comment by Abhishek-2416** [comment]
Oh my bad 
Thank you :)
